src/main/java/bio/terra/cli/service/utils/WorkspaceManagerService.java

```diff
@@ -172,20 +177,19 @@ public WorkspaceDescription createWorkspace() {
    */
   public WorkspaceDescription getWorkspace(UUID workspaceId) {
     WorkspaceApi workspaceApi = new WorkspaceApi(apiClient);
-    WorkspaceDescription workspaceWithContext = null;
     try {
       // fetch the Terra workspace object
-      workspaceWithContext = workspaceApi.getWorkspace(workspaceId);
+      WorkspaceDescription workspaceWithContext = workspaceApi.getWorkspace(workspaceId);
       String googleProjectId =
           (workspaceWithContext.getGoogleContext() == null)
```

Comment:
nit: I find Optional chains easier to read than the ternary operator.

Reply:
I don't have super strong feelings about Optional, but prefer to avoid the creation of objects unless it's strictly necessary.